testbeam_analysis/track_analysis.py

```diff
@@ -185,12 +185,23 @@ def fit_tracks(input_track_candidates_file, input_alignment_file, output_tracks_
     pixel_size : iterable of tuples
         One tuple per DUT describing the pixel dimension (column/row),
         e.g. for two FE-I4 DUTs [(250, 50), (250, 50)]. Only needed for Kalman Filter.
+    n_pixels : iterable of tuples
+        One tuple per DUT describing the number of pixels in column, row direction
+        e.g. for 2 DUTs: n_pixels = [(80, 336), (80, 336)]. Only needed for Kalman Filter.
     beam_energy : uint
         Energy of electron beam in MeV. Only needed for Kalman Filter.
     total_thickness : iterable
-        Total thickness of all DUTs in um. (Sensor + other materials)
+        Total thickness of all DUTs in um (Sensor + other materials). Only needed for Kalman Filter.
```

Comment:
Why not define a total material budged. Then we do not need to have thickness and rad. len., no?